src/utils/transform.js

```diff
@@ -536,12 +538,15 @@ export const transformValueToComponentStateFormat = (value, componentProps) => {
 						if (typeof valObj === 'object' && valObj.label && valObj.value) {
 							transformedValue.push(valObj);
 						} else if (typeof valObj === 'string') {
-							const findDataObj = data.find(item => item.label.trim() === valObj.trim());
+							const findDataObj = data.find(item => item.value.trim() === valObj.trim());
```

Comment:
@SavvyShah User would most likely pass a `label` to select the values, because that's what is visible on the UI, revert this change. 

or if you have an explanation then please do let me know.

Reply:
The CAF and QAF values are not `Array<Label>` but `Array<Value>`. Hence it doesn't work if I change it back. Or we can change the logic inside the component itself.